### DISCUSSION TITLE
Strange issue with the README file

### DISCUSSION THREAD

**Discussion by daveverwer** [soulverteam/SoulverCore#3] [General]
Hey Zac! Hope you're well.

We recently discovered an [issue with SoulverCore](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/1628) over on the package index, and after much experimentation, I think it's an issue with this GitHub repository, but I don't know how it happened or how to fix it.

If you look at the direct URL to the `README.md` file of most repositories, it renders the Markdown into HTML, [like this](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/blob/main/README.md).

However, the README file for this repo [shows the source code](https://github.com/soulverteam/SoulverCore/blob/master/README.md) for that same page. It renders fine on the project home page, but not at that URL for the README properly.

I tried reproducing this on one of my repositories, thinking it might be something like the `<p>` tags at the start of the file, but couldn't. I even tried copying the entirety of your README temporarily to see if it happened.

Did you do anything special to that file? I can't figure it out.

**Comment by zcohan** [comment]
Hey Dave,

I think the problem was that Swift was being specified as the language for all files in the repository (in the .gitattributes file). I originally set this so that Github would show the repository as 100% Swift, rather than Objective-C, but Github was probably relying on that to decide whether to display a Markdown preview or not.

Anyway, I've removed that attribute and it looks like it's working now.

**Reply by daveverwer** [reply]
Wonderful! Thanks Zac!
